src/Result.php

```diff
@@ -184,7 +184,7 @@ public function iterateAssociative(): Traversable
     }
 
     /**
-     * {@inheritDoc}
+     * @return Traversable<int,list<mixed>>
```

Comment:
This was the only error that popped up while removing the `checkMissingIterableValueType: false` setting, so I decided to fix it instead.

The `@inheritDoc` statement is nonsense because there's no parent to inherit from.